Dockerfile.latest

```diff
@@ -76,5 +76,7 @@ ENV LC_ALL=en_US.UTF-8
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-# just some smoke tests, make sure rake and YaST work properly
-RUN rake -r yast/rake -V && TERM=xterm yast2 proxy summary && rm -rf /var/log/YaST2/y2log
+# just some smoke tests, make sure rake and YaST work properly,
+# ensure there is no leftover in the working directory
+RUN rake -r yast/rake -V && TERM=xterm yast2 proxy summary && \
+  rm -rf /var/log/YaST2/y2log && rm -rf /usr/src/app/*
```

Comment:
another idea, why not run this in some tmp directory and then switch WORKDIR? so no cleaning is needed

Reply:
If I am not wrong, those files will be in the temporary directory so you will lose some space (not a big deal, but still).

Reply:
Yes, I was also thinking about it, but leaving some strange files in the image does not look nice to me anyway...

Reply:
Moreover I'd consider this is as a temporary hot fix which should be removed sooner or later (hopefully).